src/js/components/AppHealthComponent.jsx

```diff
@@ -43,6 +44,24 @@ var AppHealthComponent = React.createClass({
 
   getHealthBar: function () {
     var health = this.props.model.health;
+    var status = this.props.model.status;
+
+    if (status === AppStatus.DEPLOYING) {
+      return (
+        <div
+          className="loading-bar"
+          ref="loading-bar"
```

Comment:
IMHO `refs` are better expressed in camelCase, so we can access the `this.refs` as properties and save 2 characters (`[` and `]`) :-) 
Anyway, I think it's even better to only set the `ref` value when it's actually used, as it makes the intention clearer.